src/render.js

```diff
@@ -49,7 +49,7 @@ export function render(vnode, parentDom, replaceNode) {
 			: null;
 
 	if (isHydrating) {
-		replaceNode = excessDomChildren[0];
+		replaceNode = excessDomChildren ? excessDomChildren[0] : null;
```

Comment:
In the above we set `excessDomChildren` to null potentially so we need to check whether or not this is possible else we have to set it to null

Reply:
Oh! good catch!